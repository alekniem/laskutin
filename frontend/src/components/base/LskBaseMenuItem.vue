<template>
    <v-list-item :id="cid" link>
        <v-list-item-title>
            {{ text | trans }}
        </v-list-item-title>
    </v-list-item>
</template>

<script>
export default {
    name: "LskBaseMenuItem",

    props: {
        cid: { type: String, required: true },
        text: { type: String, required: true },
    },
};
</script>
