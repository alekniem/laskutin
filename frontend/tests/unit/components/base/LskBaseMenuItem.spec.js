import LskBaseMenuItem from "@/components/base/LskBaseMenuItem";
import { createWrapper } from "UT/lsk_test_helper";

describe("LskBaseMenuItem", () => {
    it("should match snapshot", () => {
        const wrapper = createWrapper(LskBaseMenuItem, {
            propsData: {
                cid: "TEST_CID",
                text: "TEST TEXT",
            },
        });
        expect(wrapper.html()).toMatchSnapshot();
    });
});
