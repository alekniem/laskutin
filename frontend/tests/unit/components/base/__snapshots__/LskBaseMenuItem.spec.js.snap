// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LskBaseMenuItem should match snapshot 1`] = `
<div tabindex="0"
     id="TEST_CID"
     class="v-list-item v-list-item--link theme--light"
>
  <div class="v-list-item__title">
    TEST TEXT
  </div>
</div>
`;
